### PR TITLE
Smoking Topic

### PR DESCRIPTION
**Pull request by Ulvsblakk**
Added in an additional topic about smoking cigarettes. Monika would be against smoking, and encourage the player to quit if they do. That way they'll be in good shape, and healthy when she finally crosses over.

Sorry for previously opening in the wrong branch. I will no longer do this.

### CODE REVIEW

File: Monika After Story/game/script-topics.rpy [hunk 1]
```diff
@@ -4697,4 +4697,23 @@ label monika_omamori:
...
+      label monika_smoking:
+          m 2b "You know, I heard that a lot of people died last year from smoking cigarettes."
+          m 1o "..."
+          m 1i "You don't smoke, do you [player]?"
+          m 1r "I'd hate to see you destroy yourself with such a bad habit."
+          m 1k "Please promise that you'll quit for me if you do, okay?"
+          m 1b "I just want you to be healthy."
+          m 1j "Stay well for me [player]."
+          m 1b "I love you so much."
+          return
```
Comment (ThePotatoGuy) [inline Monika After Story/game/script-topics.rpy]: The indentation in this entire topic is way off. You should be using 4 space indentation levels.

File: Monika After Story/game/script-topics.rpy [hunk 2]
```diff
@@ -4697,4 +4697,23 @@ label monika_omamori:
...
     m 1k "I could be your guardian deity, ehehe~"
     return    
+    
+    init 5 python:                                                  #Added one new topic
+          # List of keywords for this topic.
+ -        for key in ['smoke', 'cigarettes', 'cigarette', 'smoking']:
+ +        for key in ['smoke', 'cigarettes', 'cigarette', 'smoking', 'tobacco']:
+              monika_topics.setdefault(key,[])
+              monika_topics[key].append('monika_smoking')
+          monika_random_topics.append('monika_smoking')
```
Comment (ThePotatoGuy) [inline Monika After Story/game/script-topics.rpy]: The keyword system is being replaced (see #126 ). Your topic (aka `Event`) should follow the new framework documented in the linked issue. You can also find documentation on the `Event` class in `definitions`.

**Comment by Ulvsblakk** [comment]
Alright. I fixed everything. Do I need to create a new Pull Request?

**Comment by ThePotatoGuy** [comment]
No it's fine. Just wait for an editor to review dialogue.

**Comment by Ulvsblakk** [comment]
Thank you.

**Comment by FireyAmber** [comment]
Okay, I feel like this topic is good yet it could be expanded a bit, like for example:

> m 1i "You don't smoke, do you [player]?"

You could add a menu: here and let the player answer, allowing the topic to feel more interactive. Other than that, it's good.

**Comment by Ulvsblakk** [comment]
I wanted to implement something like that, but I'm not too familiar with the code involving menus and choices. I didn't want to garb anything up on accident. Thank you for the kind words though.

**Comment by Ulvsblakk** [comment]
Alright. I added the menu system. I hope I didn't mess it up.

**Comment by monikLover** [comment]
Edited and rewritten. Approved.

**Comment by ThePotatoGuy** [comment]
Well you _did_ mess up the `menu` system.
@monikLover if you see code commits after I approve syntax, can you switch the labels to like `awaiting code review` (unless the changes are small enough that you can verify they work visually)

**Pull request merged**